SplunkAppForWazuh/appserver/static/js/controllers/overview/general/overviewGeneralCtrl.js

```diff
@@ -81,7 +82,30 @@ define([
           
         } else {
           this.scope.wzMonitoringEnabled = true
-          this.vizz.push(new AreaChart(`agentStatusHistory`,`index=wazuh-monitoring-3x status=* | timechart span=1h count by status usenull=f`,`agentStatus`))
+
+          //Filters for agents Status
+          this.backupFilters = this.currentDataService.getFilters()
+          if (Object.keys(this.backupFilters[0])[0] == 'manager.name'){

```

Comment:
I think we can optimize this process by avoiding the backup of the filters and restoring them later:
- As we want to include new filters as string, we can build a custom string variable with the wanted filters. Just check the content of the current filters and select the wanted values to append it to the new string.